db/seeds.rb

```diff
@@ -2115,4 +2115,10 @@
   description: "A musical in which a modern day Romeo and Juliet are involved in New York street gangs. 
 On the harsh streets of the upper west side, two gangs battle for control of the turf. 
 The situation becomes complicated when a gang members falls in love with a rival's sister."
-)
\ No newline at end of file
+)
+test_user = User.create(email:'testpass@example.com', password: 'valid_password', password_confirmation: 'valid_password')
+
+review1 = Review.create(rating: "5", body: "Review1", movie_id: movie1.id, user_id: test_user.id)
+review2 = Review.create(rating: "4", body: "Review2", movie_id: movie1.id, user_id: test_user.id)
+review3 = Review.create(rating: "5", body: "Review3", movie_id: movie2.id, user_id: test_user.id)
+review4 = Review.create(rating: "2", body: "Review4", movie_id: movie2.id, user_id: test_user.id)
```

Comment:
lets use some slightly more real data here. Not needed for the user, but lets do it for our reviews and movies